rpcs/src/client/jsonrpsee_impl.rs

```diff
@@ -60,9 +60,27 @@ impl RpcClientT for Client {
             };
 
             let stream = stream
-                .map_err(|e| RpcError::ClientError(Box::new(e)))
+                .map_err(|e| Error::Client(Box::new(e)))
                 .boxed();
             Ok(RawRpcSubscription { stream, id })
         })
     }
 }
+
+/// Convert a JsonrpseeError into the RPC error in this crate.
+/// The main reason for this is to capture user errors so that
+/// they can be represented/handled without casting. 
+fn error_to_rpc_error(error: JsonrpseeError) -> Error {
```

Comment:
same here, can we change it to an into/from impl instead?

Reply:
Done!